mmv1/third_party/terraform/services/modelarmorglobal/resource_model_armor_floorsetting_test.go

```diff
@@ -0,0 +1,167 @@
+package modelarmorglobal_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccModelArmorGlobalFloorsetting_basic(t *testing.T) {
+
+	basicContext := map[string]interface{}{
+		"location": "global",
+		"parent":   "projects/ci-test-project-188019",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccModelArmorGlobalFloorsetting_basicContext(basicContext),
+			},
+			{
+				ResourceName:      "google_model_armor_floorsetting.floorsetting-basic",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccModelArmorGlobalFloorsetting_basicContext(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_model_armor_floorsetting" "floorsetting-basic" {
+  parent      = "%{parent}"
```

Comment:
```suggestion
  parent      = "project/%{project_id}"
```

Similar for the rest of the handwritten tests